script.js

```diff
@@ -0,0 +1,256 @@
+      else if (event.code == "MetaRight") element[1].classList.add('keypress');
+      else if (event.code == "AltLeft") element[0].classList.add('keypress');
+      else if (event.code == "AltRight") element[1].classList.add('keypress');
+    }
+    if (event.keyCode == 16) {
+      let langKeys;
+      isEng == 1 ? langKeys = engKeysShift : langKeys = rusKeysShift;
+      let i = 0;
+      let keys = document.querySelectorAll('#keyboard div');
+      keys.forEach((el) => {
+        el.innerText = langKeys[i];
+        i++;
+      });
+    }
+  }
+
+  document.addEventListener('keydown', (event) => {
+    keydown(event);
+  });
+  document.getElementById('keyboard').childNodes.forEach((e) => {
+    e.addEventListener('mousedown', (event) => {
+      let element = document.querySelectorAll('div[data="' + `${event.target.getAttribute('data')}` + '"]');
+      if (element.length == 1) {
+        element = element[0];
+        element.classList.add('keypress');
+      } else {
+        element = event.target;
+        element.classList.add('keypress');
+      }
+    });
+  })
+  document.getElementById('keyboard').childNodes.forEach((e) => {
+    e.addEventListener('mouseup', (event) => {
+     let element = document.querySelectorAll('div[data="' + `${event.target.getAttribute('data')}` + '"]');
+     if (element.length == 1) {
+       element = element[0];
+       element.classList.remove('keypress');
+       if (element.innerText == 'Backspace') {
+         output = output.substr(0, output.length-1);
+       } else if (element.innerText == 'Enter') {
+         output += '\n';
+       } else if (element.innerText == '') {
+         output += ' ';
+       } else if (element.innerText == 'Tab') {
+         output += '   ';
+       } else if (element.innerText == 'Caps Lock') {
+         output += '';
+       } else {
+         output += element.innerText;
+       }
+       document.querySelector('textarea').value = output;
+     } else {
+       element = event.target;
+       element.classList.remove('keypress');
+     }
+     let keyCode = event.target.getAttribute('data');
+     if (keyCode == 20) {
+       caps = !caps;
+       let langKeys;
+       isEng == 1 ? langKeys = engKeysUpperCase : langKeys = rusKeysUpperCase;
+       let i = 0;
+       let keys = document.querySelectorAll('#keyboard div');
+       if (caps) {
+         element.classList.add('keypress');
+         keys.forEach((el) => {
+           el.innerText = langKeys[i];
+           i++;
+         })
+       } else {
+         isEng == 1 ? langKeys = engKeys : langKeys = rusKeys;
+         element.classList.remove('keypress');
+         keys.forEach((el) => {
+           el.innerText = langKeys[i];
+           i++;
+         });
+       }
+     }
+   });
+  });
+
+  document.addEventListener('keyup', function(event) {
+    if (event.code == 'ShiftLeft') leftShift = false;
+    else if (event.code == 'AltLeft') {
+      if (leftShift) {
+        isEng == 1 ? isEng = 0 : isEng = 1;
```

Comment:
You can change your condition like this -
isEng = Number(!isEng);
it will be 1 when isEng is 0 and 0 when isEng is 1

Reply:
edited